channeldb/channel.go

```diff
@@ -3252,6 +3252,96 @@ const (
 	Abandoned ClosureType = 5
 )
 
+// LocalForceCloseInitiator is a type that gives information about what led to a
+// channel being force closed locally.
+type LocalForceCloseInitiator string
```

Comment:
I think since there are plans to include the remote force close reason in the future and this just specifically states what initiated the local force close, the name given in this PR is more befitting.

Reply:
Ref: https://github.com/lightningnetwork/lnd/blob/0df507eca09a545e6fc41ba5eb243b029daf5a14/htlcswitch/failure.go#L30-L39 to the structure of `LinkError` type.